make systemd optional and always install systemd_files because it provides tmpfiles
--- a/meson.build
+++ b/meson.build
@@ -186,7 +186,7 @@ summary({'sendmmsg': sendmmsg,
 ### Systemd; most is in systemd/meson.build
 systemd_files = get_option('systemd_files')
 systemd_legacy_units = get_option('systemd_legacy_units')
-libsystemd = dependency('libsystemd', required: systemd_files == 'enabled')
+libsystemd = dependency('libsystemd', required: get_option('systemd'))
 summary({'libsystemd support': libsystemd.found(),
          'files': systemd_files == 'enabled',
         }, section: 'Systemd', bool_yn: true)
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -227,3 +227,10 @@ option(
   value: 'auto',
   description: 'add rpath to the knot resolver executables',
 )
+
+option(
+  'systemd',
+  type: 'feature',
+  value: 'auto',
+  description: 'systemd watchdog support',
+)
